zio-http/src/main/scala/zio/http/model/headers/Structured.scala

```diff
@@ -0,0 +1,75 @@
+package zio.http.model.headers
+
+import zio.Chunk
+
+import scala.util.Try
+
+object Structured {
```

Comment:
Maybe this should be called `HeaderValues` and located in `zio.http.model`, instead? I think I'd like to delete the headers package eventually.

Reply:
renamed to `HeaderValue` as `HeaderValues` is already used ...